packages/providers/upload-aws-s3/lib/__tests__/getBucketFromUrl.test.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const { getBucketFromUrl } = require('../utils');
+
+describe('Test for URLs', () => {
+  test('Virtual hosted style', async () => {
+    const url = 'https://bucket.s3.us-east-1.amazonaws.com/img.png';
+    const { bucket } = getBucketFromUrl(url);
+    expect(bucket).toEqual('bucket');
+  });
+
+  describe('Path style', () => {
+    test('No key', async () => {
+      const url = 'https://s3.us-east-1.amazonaws.com/bucket';
+      const { bucket } = getBucketFromUrl(url);
+      expect(bucket).toEqual('bucket');
+    });
+
+    test('With trailing slash', async () => {
+      const url = 'https://s3.us-east-1.amazonaws.com/bucket/';
+      const { bucket } = getBucketFromUrl(url);
+      expect(bucket).toEqual('bucket');
+    });
+
+    test('With key', async () => {
+      const url = 'https://s3.us-east-1.amazonaws.com/bucket/img.png';
+      const { bucket } = getBucketFromUrl(url);
+      expect(bucket).toEqual('bucket');
+    });
+  });
+
+  test('Virtual hosted style', async () => {
```

Comment:
Is this test description correct? it is the same as the one at L6

Reply:
oops u are right it should be another one.